src/constants/index.ts

```diff
@@ -9,22 +9,43 @@ export const SUBGRAPH_ENDPOINT = process.env.REACT_APP_SUBGRAPH_ENDPOINT as stri
 export const FE_VERSION = '0.0.1'
 export const SC_VERSION = '0.0.1'
 export const NETWORK_CONTEXT_NAME = 'Network'
-export const SUPPORTED_CHAIN_IDS = [100]
 export enum CHAIN_ID {
-  MAINNET = 1,
   RINKEBY = 4,
   XDAI = 100,
 }
-export const XDAI_CHAIN_PARAMETER = {
-  chainId: '0x64',
-  chainName: 'xDai',
-  nativeCurrency: {
-    name: 'xDai',
-    symbol: 'xDai',
-    decimals: 18,
+export const SUPPORTED_CHAIN_IDS = [CHAIN_ID.RINKEBY, CHAIN_ID.XDAI]
+
+export const SUPPORTED_CHAINS = {
+  [CHAIN_ID.RINKEBY]: {
+    id: CHAIN_ID.RINKEBY,
+    name: 'Rinkeby Testnet',
+    parameters: {
+      chainId: '0x4',
+      chainName: 'Ethereum',
+      nativeCurrency: {
+        name: 'Ethereum',
+        symbol: 'ETH',
+        decimals: 18,
+      },
+      rpcUrls: ['https://rinkeby.infura.io/v3'],
+      blockExplorerUrls: ['https://rinkeby.etherscan.com'],
+    },
+  },
+  [CHAIN_ID.XDAI]: {
+    id: CHAIN_ID.XDAI,
+    name: 'xDai Chain',
+    parameters: {
+      chainId: '0x64',
+      chainName: 'xDai',
+      nativeCurrency: {
+        name: 'xDai',
+        symbol: 'xDai',
+        decimals: 18,
+      },
+      rpcUrls: ['https://rpc.xdaichain.com/'],
+      blockExplorerUrls: ['https://blockscout.com/xdai/mainnet'],
+    },
```

Comment:
It might be good in the future to create a custom hook that would be able to fetch these constants for you.